CHANGELOG.md

```diff
@@ -31,6 +31,9 @@
 - Bump CLI from v2.50.2 to v2.51.1 ([#5053](https://github.com/getsentry/sentry-react-native/pull/5053), [#5058](https://github.com/getsentry/sentry-react-native/pull/5058))
   - [changelog](https://github.com/getsentry/sentry-cli/blob/master/CHANGELOG.md#2511)
   - [diff](https://github.com/getsentry/sentry-cli/compare/2.50.2...2.51.1)
+- Bump Bundler Plugins from v4.0.2 to v4.1.0 ([#5062](https://github.com/getsentry/sentry-react-native/pull/5062))
+  - [changelog](https://github.com/getsentry/sentry-javascript-bundler-plugins/blob/main/CHANGELOG.md#410)
+  - [diff](https://github.com/getsentry/sentry-javascript-bundler-plugins/compare/4.0.2...4.1.0)
```

Comment:
There is the same changelog below but mentioning a newer version, lets keep the version below.
```suggestion
```